Pipfile

```diff
@@ -12,6 +12,10 @@ whitenoise = "6.2.0"
 gunicorn = "20.1.0"
 djangorestframework = "3.13.1"
 drf-yasg = "1.20.0"
+typing-extensions = "4.2.0"
```

Comment:
No need for this python has an inbuilt typing library

Reply:
Okay